app/assets/stylesheets/application.scss

```diff
@@ -14,6 +14,7 @@
  *= require_self
  */
 
+@import "external-links";
```

Comment:
Looks like this is not necessary as `require_tree .` already imports it. (`require_tree .` => require all the files and directories recursively from this file's directory)

We might decide to remove `require_tree .` if we want to use SCSS variables across multiple files though.

Reply:
Works without includes of `hero` and `external-links` so I've removed both of 'em